openedx/adg/admin_dashboard/applications/views.py

```diff
@@ -101,6 +104,36 @@ def setup(self, request, *args, **kwargs):
         super().setup(request, *args, **kwargs)
         self.application_type = kwargs.get('application_type')
 
+    def get(self, request, *args, **kwargs):
+        """
+        Attaches csv report for download with response.
+
+        Returns:
+            HttpResponse: Response with grades report.
+        """
+        if not request.GET.get('download_report') or request.GET.get('download_report') == 'False':
```

Comment:
We do not need to get the value twice from the request_dict.
Perhaps something like `request.GET.get('download_report', False) == False` will work for us here.